src/index.js

```diff
@@ -11,15 +11,15 @@ import * as serviceWorker from './serviceWorker'
 
 ReactDOM.render(
   <Router>
-    <UserProvider>
-      <SettingsProvider>
+    <SettingsProvider>
+      <UserProvider>
         <ViewProvider>
           <SessionProvider>
             <App />
           </SessionProvider>
         </ViewProvider>
-      </SettingsProvider>
-    </UserProvider>
+      </UserProvider>
+    </SettingsProvider>
```

Comment:
I'm really happy that switching these around worked like this! Great job figuring this out 🤘🏼